frontend/lib/src/components/elements/Markdown/Markdown.tsx

```diff
@@ -35,7 +37,15 @@ export default function Markdown({
   width,
   element,
 }: MarkdownProps): ReactElement {
+  const { hostConfig } = React.useContext(LibContext)
+  if (hostConfig.disableUnsafeHtmlExecution && element.allowHtml) {
```

Comment:
Small bug: This should be:
```
hostConfig.disableUnsafeHtmlExecution && !element.allowHtml
```

Will change and commit.

Reply:
Hmm, I think it needs to be `element.allowHtml`, since we want to prevent the rendering only if `element.allowHtml==True`.